### ISSUE TITLE
Recherche: Text "Brand" is in English on the French page

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#164]
**Page:** Recherche
**URL:** https://rsams.dev.cloud.openplus.ca/fr/recherche/site?f%5B1%5D=brand%3A780 and https://recalls-rappels.canada.ca/fr/recherche/site?f%5B1%5D=brand%3A920
**User Agent:** Firefox (v99)
**Language:** French

**Issue Description:** Text "Brand" in the button label is in English on the French page.

![image](https://user-images.githubusercontent.com/12256067/162457673-90488c87-0f35-4495-ada2-26b0e322f23c.png)

**Steps to reproduce:**

1. Go to the above URL
2. Visual scanning

**Expected results:** The text "Brand" should be in French.

**Comment by echouin** [comment]
Reported to OP, awaiting fix.

**Comment by echouin** [comment]
Hi @sarasuri this has been completed and is now live on the website: Example: https://recalls-rappels.canada.ca/fr/recherche/site?f%5B1%5D=brand%3A1233

![image](https://user-images.githubusercontent.com/88099454/192033070-80c6ef43-7105-4e7e-9c3f-a0607d4b8384.png)

**Comment by sarasuri** [comment]
Verified (Sept 27, 2022) on PROD, the issue is fixed. Not able to verify on DEV as it wasn't present at the time of reviewing.
PROD:https://recalls-rappels.canada.ca/fr/recherche/site?f%5B1%5D=brand%3A1211

![image](https://user-images.githubusercontent.com/12256067/192602967-1157e042-48f7-4bff-b055-622922ea80c8.png)

Closing the issue.
